routes/userRegistration.js

```diff
@@ -9,5 +9,8 @@ router.post("/", userRegistrationController.createUser);
 router.get("/:username", userRegistrationController.userByusername);
 router.put("/:username", requireSignin, userRegistrationController.update);
 router.delete("/:username", userRegistrationController.deleteUser);
+//for disabled users
+router.put('/disable/:username', userController.disableUser);
```

Comment:
This is being removed because the route already exists, quite literally above it.
```suggestion
```